app/src/main/res/layout/fragment_user_profile.xml

```diff
@@ -119,11 +119,13 @@
         android:id="@+id/log_out_button"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
-        android:layout_marginTop="60dp"
+        android:layout_marginTop="30dp"
+        android:layout_marginBottom="50dp"
```

Comment:
The bug is still there with these new values

Reply:
I put a bottom constraint, but if it still doesn't work, I leave you this bug to you because I can't emulate devices on my pc.

Reply:
Bug still there, I'll look into it by leaving #63 open, for now I'll approve the pull request.